clicache/integration-test/SerializationTestsN.cs

```diff
@@ -92,19 +92,19 @@ public void CreateRegionForOT(string locators)
     {
       CacheHelper.CreateTCRegion2<object, object>(RegionNames[0], true, false,
         null, locators, false);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType.CreateDeserializable);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType22.CreateDeserializable);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType4.CreateDeserializable);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType2.CreateDeserializable);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType42.CreateDeserializable);
-      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType43.CreateDeserializable);
+      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType.CreateDeserializable, 0);
+      CacheHelper.DCache.TypeRegistry.RegisterType(OtherType22.CreateDeserializable, 0x8C0);
```

Comment:
What is the purpose of the class id in hex?

Reply:
No purpose.

Reply:
Legacy "id"

Reply:
i.e. that was the ClassId that the `OtherType22` class always contained, we simply moved that info along to the new api

Reply:
Ok, hex just made it look "special".